src/tickit/adapters/epicsadapter/adapter.py

```diff
@@ -30,14 +30,18 @@ class OutputRecord:
 
 
 class EpicsAdapter(Adapter):
-    """An adapter implementation which acts as an EPICS IOC."""
+    """An adapter implementation which acts as an EPICS IOC.
 
-    def __init__(self, db_file: str, ioc_name: str) -> None:
+    This is optionally initialised from an EPICS database (db) file
+    but can be customised in code by implementing on_db_load.
+    """
+
+    def __init__(self, ioc_name: str, db_file: Optional[str] = None) -> None:
```

Comment:
For the parameter type annotation, which is better out of `Optional[str]` or `str | None`?

Reply:
`str | None` but it's only available in 3.10+ so I think we need to keep using `Optional[str]` for now